sql_composer/sql_composer.py

```
@@ -0,0 +1,185 @@
+            self.translator.val_to_sql(column_map[col_name], value)
+            for col_name, value in key_values.items()
+            if column_map.get(col_name, None) is not None
+        ]
+
+        stmt = f"""
+        INSERT INTO {self.table.name}
+        ({",".join(col_names)})
+        VALUES
+        ({",".join(col_values)}) 
+        ;
+        """
```

Ruff (0.14.7)

Possible SQL injection vector through string-based query construction

(S608)